lib/tasks/jobs.rake

```diff
@@ -0,0 +1,17 @@
+namespace :db do
+  desc "Schedule all cron jobs"
+  task schedule_jobs: :environment do
+    glob = Rails.root.join("app", "jobs", "**", "*_job.rb")
+
+    Dir.glob(glob).each { |f| require f }
+    CronJob.subclasses.each { |job| job.schedule }
```

Comment:
With other cron extensions I've seen (like [Resque Scheduler](https://github.com/resque/resque-scheduler)), they've used a YAML file to specify scheduled jobs. I like this subclass approach, but I'm not sure if it's a bit too opaque.